services/movie.py

```diff
@@ -0,0 +1,29 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    set_ = Movie.objects.all()
+    if genres_ids:
+        set_ = set_.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        set_ = set_.filter(actors__id__in=actors_ids)
+    return set_
+
+
+def get_movie_by_id(movie_id: int = None) -> Movie:
```

Comment:
`None` is not a valid value for retrieving a record from the database. If you don't pass movie_id, the method will try to execute Movie.objects.get(id=None), which will result in an error because None is not a valid id.